### PR TITLE
quick-search.component | rebranding 2

### PR DESCRIPTION
**Pull request by mhigbyflowers**
The only files to that belong to this PR are:
quick-search.component.html
quick-search.component.css
quick-search.component.ts

### CODE REVIEW

**Comment by mhigbyflowers** [comment]
ci/circleci: public-ui-build-test is failing but is resolved in this pull request #38

File: public-ui/src/app/views/quick-search/quick-search.component.ts [hunk 1]
```diff
@@ -10,6 +10,8 @@ import { Observable } from 'rxjs/Rx';
 import { ISubscription } from 'rxjs/Subscription';
 import {DataBrowserService} from '../../../publicGenerated/api/dataBrowser.service';
 import {DomainInfosAndSurveyModulesResponse} from '../../../publicGenerated/model/domainInfosAndSurveyModulesResponse';
```
Comment (dolbeew) [inline public-ui/src/app/views/quick-search/quick-search.component.ts]: You could combine these two imports with a shorter path `import {DataBrowserService, DomainInfosAndSurveyModulesResponse} from 'publicGenerated';`. I know these weren't part of your PR, but my IDE pointed it out.

File: public-ui/src/app/views/quick-search/quick-search.component.css [hunk 2]
```diff
@@ -1,38 +1,159 @@
...
+    margin: 0.6em 0.5em 0.1em;
+}
+
+.result-heading {
+    margin-left: 0;
+    margin-bottom: 8px;
+    font-family: unit, open-sans, sans-serif;
+    padding: 0;
+    color: #302C71;
+    font-family: unit, open-sans, sans-serif;
```
Comment (dolbeew) [inline public-ui/src/app/views/quick-search/quick-search.component.css]: Duplicate `font-family` property

File: public-ui/src/app/styles/page.css [hunk 3]
```diff
@@ -1,197 +1,227 @@
...
-    padding: 1rem;
-    padding-top: .5rem;
-    width: 100%;
-    background-color: #FFFFFF;
-    border-radius: 4px;
-    box-shadow: 0 4px 12px 0 rgba(0,0,0,0.15);
-    position: relative;
+#db-title {
+  text-align: center;
+  width: calc((100%/12)- 8px);
```
Comment (dolbeew) [inline public-ui/src/app/styles/page.css]: This showed as invalid property value in my browser. Add a space before the `-` and it seems to work.

File: public-ui/src/app/views/quick-search/quick-search.component.css [hunk 4]
```diff
@@ -1,38 +1,159 @@
...
+    flex-grow: 1;
+    font-size: 20px;
 }
-/* result card */
-.white-card-box.search-result {
-    height: 100%;
-    width: 100%;
-    margin-top: 1rem;
+
+.search .sarch-bar {
```
Comment (dolbeew) [inline public-ui/src/app/views/quick-search/quick-search.component.css]: Looks like a typo

**Review by dolbeew — COMMENTED** [review]
Just a few comments. Also one display nit: the result boxes in the current quick search fit four wide on most screens while the new layout fits three unless you go >1770px. 
screen shot 2019-02-26 at 12 47 23 pm
Is this intended for the new design?

**Review by dolbeew — APPROVED** [review]
Looks good.

**Pull request merged**